community/kernel/src/main/java/org/neo4j/kernel/api/proc/ProcedureSignature.java

```diff
@@ -182,9 +202,9 @@ public int hashCode()
     @Override
     public String toString()
     {
-        String strInSig = inputSignature == null ? "..." : iterableToString( typesOf( inputSignature ), ", " );
-        String strOutSig = outputSignature == null ? "..." : iterableToString( typesOf( outputSignature ), ", " );
-        return String.format( "%s(%s) : (%s)", name, strInSig, strOutSig );
+        String strInSig = inputSignature == null ? "..." : iterableToString( inputSignature, ", " );
```

Comment:
you do not need to have separate method `iterableToString` please use `Iterables.toString`